src/sentry/static/sentry/app/components/smartSearchBar/index.tsx

```diff
@@ -1034,6 +1034,7 @@ class SmartSearchBar extends React.Component<Props, State> {
 
       // then update the autocomplete box with new contextTypes
       this.updateAutoCompleteItems();
+      this.props.onChange?.(newQuery, {} as any);
```

Comment:
i've seen us mock events this way before, it doesn't feel good

Reply:
You could create a [MouseEvent](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/MouseEvent) object and pass that along instead.